cosmos/models/board.py

```diff
@@ -0,0 +1,35 @@
+from django.contrib.auth.models import Group
+from django.db import models
+
+
+class Board(models.Model):
+    """
+    Extension of Django Group modelt to store extra data of the board.
+
+    - `name`, `permissions`: self-explanatory
+
+    To get all groups of a user you can do this:
+
+    - user.groups.all()
+
+    To get all users of a group:
+
+    - group.user_set.all()
+    """
+
+    group = models.OneToOneField(Group, on_delete=models.CASCADE)
+    description = models.TextField(blank=True)
+    period_from = models.DateField()
+    period_to = models.DateField()
+    pretix_organizer = models.CharField(max_length=20)
```

Comment:
Is there a way to store the position the individual has in the board / committee?

Reply:
This model is to label a person as part of the board. However, I haven't considered how to label a certain individual as a certain position in the board/committee

Reply:
@bcvandendool refer to comment above, I posted it under the wrong comment.